tests/garage/tf/baselines/test_deterministic_mlp_baseline_with_model.py

```diff
@@ -0,0 +1,97 @@
+import pickle
+from unittest import mock
+
+from nose2.tools.params import params
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.baselines import DeterministicMLPBaselineWithModel
+from garage.tf.envs import TfEnv
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyBoxEnv
+from tests.fixtures.regressors import SimpleMLPRegressor
+
+
+class TestDeterministicMLPBaselineWithModel(TfGraphTestCase):
+    @params([1], [2], [1, 1], [2, 2])
+    def test_fit(self, obs_dim):
+        box_env = TfEnv(DummyBoxEnv(obs_dim=obs_dim))
+        with mock.patch(('garage.tf.baselines.'
+                         'deterministic_mlp_baseline_with_model.'
+                         'DeterministicMLPRegressorWithModel'),
+                        new=SimpleMLPRegressor):
+            gmb = DeterministicMLPBaselineWithModel(env_spec=box_env.spec)
```

Comment:
minor thing, is `dmb` a better name, I guess `gmb` stands for GaussianMLPBaseline?

Reply:
Yes, I will change it to `cmb`.